### ISSUE TITLE
Poker: Estimate: GitHub Label Input Template

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#5364]
When playing planning poker in Jira, users can push a numeric value to a particular field.
For example, if that field is called "Story Point Estimate" then a user can push the value "5" to it.

With GitHub, we don't have access to fields like that.
Instead, a common practice is to use labels.
For example, if we were ranking bugs, labels might look like this:
![Screenshot from 2021-09-03 12-45-45](https://user-images.githubusercontent.com/5514175/132058512-1c51362f-9a6a-43c3-abc9-197219f90321.png)

These labels follow a standard pattern. They have a prefix, value, suffix.
Examples:
- `SP: 5`
- `Estimate: 5 points`
- `P0 - Severe`

I _think_ the best way to handle this is to give the User an input field & let them use a wildcard for where the points should go.

Possible Options:
- `SP: {{points}}`
- `Estimate: % points`
- `Prefix: '', Suffix: ' - Severe'` (2 input fields, this is my least favorite option)

My thought is to put this inline in the dropdown menu, or maybe they click "Edit Label" and it opens up a modal where they can edit it there:
![Screenshot from 2021-09-03 12-35-53](https://user-images.githubusercontent.com/5514175/132058901-e1af2db3-7214-4fcb-ad82-708bae38b490.png)

AC
- [ ] A figma design allowing users to write a template to convert their story point estimate to a GitHub Label

Estimate: 8 hours? Not a designer :shrug: 

@ackernaut @enriquesanchez

**Comment by enriquesanchez** [comment]
@mattkrick 

[Here's a 4-min Loom with an initial proposal for this](https://www.loom.com/share/999072a1d6ba4c8ea4641f58deeea5ad). 

Let me know what you think. I may have missed a thing or two.

[The Figma file is here](https://www.figma.com/file/MbaWF1xjjD038579oTVp5b/GH-label-input-template?node-id=0%3A1), but here are a couple of screen for reference:

![image](https://user-images.githubusercontent.com/3276087/133342915-180a5324-6ee4-40f3-b9b8-b1330eaa264e.png)
![image](https://user-images.githubusercontent.com/3276087/133342934-652ac694-a052-4f29-9434-f9eab34a7f3d.png)

**Comment by mattkrick** [comment]
oh this is looking _clean_. thanks for the fast turnaround!
i like the modal & wildcard look!

- i think the only change is we don't need to support multiple labels. 1 dimension (e.g. "Story Points" or "Estimated Value") will only have 1 label & that won't often change, so there's no need for the multiple labels or titles. So i think the dropdown will look something like this:
- `Estimate: # points`
-  `As Comment`
- `Do not update`

1 other question-- what if they want an actual hash sign `#` in their label instead of the wildcard?

**Comment by enriquesanchez** [comment]
> So i think the dropdown will look something like this:
> Estimate: # points
> As Comment
> Do not update

@mattkrick  Oh, so they can push the value both as a label or a comment?

>what if they want an actual hash sign # in their label instead of the wildcard?

That's a good point. Can you think of another character we can use that could also be interpreted as a wildcard but maybe less common? Or maybe we offer a way to escape the character?

**Comment by mattkrick** [comment]
> Oh, so they can push the value both as a label or a comment?

Yep! Just one or the other. If they choose to push to a label, they get to create the "label template"

> what if they want an actual hash sign # in their label instead of the wildcard?

The most common usage I've seen around the internet (e.g. segment, slack) is the use of handlebars (AKA mustaches): `{{ foo }}`

 This would give us the ability to use multiple variables, which might be nice.
For example, let's say the team is doing a "Weighted Shortest Job First" template. Each issue has 2 estimates: `Story Points`  AKA how long it takes, and `Story Value` AKA how much customers want it.

Going with that example, let's say the team votes that the story points are 10 and the value is 5. Therefore the weighted shortest job first score would be 5 / 10 = `0.5`

We could have the label template be `{{ dimension }}: {{ points }}` which would turn into `Story Points: 10`, `Story Value: 5`.

My thinking is that In the future, in the "label template" modal, maybe we include an additional input if the template has more than 1 dimension. Then they could do things like `Score: {{ p2 / p1 }} points` and it would turn into `Score: 0.5 points`. I know this is a lot of text without any drawings. let's hop on a call this week!

**Comment by enriquesanchez** [comment]
Ok, here's an update based on our discussion:

![imagen](https://user-images.githubusercontent.com/3276087/134075767-7eae3900-5d4e-49da-b832-e40549bb38c4.png)

Clicking on the edit icon next to 'As a label' opens the label template modal:

![imagen](https://user-images.githubusercontent.com/3276087/134076370-dcfbe058-d474-43ff-90a9-0d1ed5ba989e.png)


Users can both type in the wildcard `{#}`, or click on the wildcard chip to have it added to the input field.
